doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3550,6 +3550,23 @@ Trusted Firmware-M (TF-M)
 
 The issues in this section are related to the TF-M implementation in the |NCS|.
 
+.. rst-class:: v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0
+
+NCSDK-22629: TF-M may fail to reset using nrfjprog versions greater or equal to 10.20.0 on nRF9160 platforms
+  Issuing a reset command using nrjrpog for nRF9160 platforms using the following command may fail to successfully complete and can cause a TF-M core panic:
+  .. code-block::
+
+     nrfjprog -f nrf91 --reset
+
+  **Workaround:** Don't use the --reset option, use --pinrset or --debugreset instead:
```

Comment:
```suggestion
  **Workaround:** Don't use the --reset option, use --pinreset or --debugreset instead:
```

Reply:
Here, --pinreset needs to be fixed still.

Should use \`\`--pinreset\`\` also i think.